build.gradle

```diff
@@ -195,6 +206,24 @@ project(':aeron-samples') {
     build.doLast {
         tasks.shadowJar.execute()
     }
+
```

Comment:
I think these tasks clutter up the build. It's not too bad to execute java at the command line and bring in a tools.jar instead. Somewhat like we do with samples.